src/Impostor.Server/Net/Matchmaker.cs

```diff
@@ -62,7 +62,7 @@ public async ValueTask StopAsync()
         private async ValueTask OnNewConnection(NewConnectionEventArgs e)
         {
             // Handshake.
-            HandshakeC2S.Deserialize(e.HandshakeData, out var clientVersion, out var name, out _);
+            HandshakeC2S.Deserialize(e.HandshakeData, out var clientVersion, out var name, out _, out _, out _);
```

Comment:
Language and chat mode api in client?

Reply:
would be cool, but I consider adding new API's out of scope for version bump PR's because I want these to land as soon as possible.

We can bikeshed API's later, you're hereby invited to take a first shot at adding internalization support to Impostor.